### PR TITLE
Dev | update login and search customer, route

### PR DESCRIPTION
**Pull request by tiepnvbn**

### CODE REVIEW

**Review by tuan280195 — CHANGES_REQUESTED** [review]
Why all the changes are out-of-date?

File: src/components/CustomerSearch.js [hunk 1]
```diff
@@ -125,51 +155,39 @@ const CustomerSearch = () => {
...
-        ) : null}
-        <Grid
-          item
-          xs={12}
-          style={{ display: "flex", justifyContent: "center" }}
-        >
-          <FormButton itemName="Search" onClick={handleClickSearch} />
-        </Grid>
-      </Grid>
-
```
Comment (tiepnvbn) [inline src/components/CustomerSearch.js:172]: THIS BLOCK IS CORRECT

File: src/components/CustomerSearch.js [hunk 2]
```diff
@@ -79,27 +111,25 @@ const CustomerSearch = () => {
...
     return (
-      <ul id="results" className="search-results" style={{ marginTop: 10 }}>
-        {listItem && listItem[0].id != null ? (
+      <ul id="results" className="search-results">
+        {listItem && listItem.length > 0 ? (
           listItem.map((item, index) => {
             return (
               <>
                 <li className="search-result" key={item + "-" + index}>
-                  <Truncate str={item.customerName} maxLength={20} />
```
Comment (tiepnvbn) [inline src/components/CustomerSearch.js:102]: CORRECT

File: src/components/CustomerSearch.js [hunk 3]
```diff
@@ -79,27 +111,25 @@ const CustomerSearch = () => {
...
+    if (item == "customerName") newData.customerName = event.target.value;
     else newData.phoneNumber = event.target.value;
 
     setData(newData);
   };
 
   const Results = () => {
     return (
-      <ul id="results" className="search-results" style={{ marginTop: 10 }}>
-        {listItem && listItem[0].id != null ? (
```
Comment (tiepnvbn) [inline src/components/CustomerSearch.js:97]: CORRECT

File: src/components/CustomerSearch.js [hunk 4]
```diff
@@ -79,27 +111,25 @@ const CustomerSearch = () => {
 
   const handleClickSearch = async (e) => {
     await getCustomers(e);
-    setCondition({ width: "1000px", xs: 6 });
```
Comment (tiepnvbn) [inline src/components/CustomerSearch.js:82]: CORRECT

File: src/components/CustomerSearch.js [hunk 5]
```diff
@@ -65,11 +99,9 @@ const CustomerSearch = () => {
     setLoading(true);
     try {
       // call API Delete
-      var deleteURL = "https://localhost:7265/api/Customer/" + deleteItem.id;
-      await axios.delete(deleteURL).then(async (res) => {
-        await getCustomers();
-        setShowAlert(false);
```
Comment (tiepnvbn) [inline src/components/CustomerSearch.js:71]: CORRECT

File: src/components/CustomerSearch.js [hunk 6]
```diff
@@ -47,16 +77,20 @@ const CustomerSearch = () => {
         });
       });
       setListItem(result); /*set result list item here*/
+      setShowList(true);
     } catch (error) {
       console.log(error);
     }
 
+    setShowList(true);
```
Comment (tiepnvbn) [inline src/components/CustomerSearch.js:85]: REMOVE

File: src/components/CustomerSearch.js [hunk 7]
```diff
@@ -47,16 +77,20 @@ const CustomerSearch = () => {
         });
       });
       setListItem(result); /*set result list item here*/
+      setShowList(true);
```
Comment (tiepnvbn) [inline src/components/CustomerSearch.js:80]: REMOVE

**Review by tuan280195 — APPROVED** [review]

**Pull request merged**